components/bl-backdrop-component/component.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Also create a README.md for component, you eecould see how it may looks there
https://github.com/Backendless/ui-builder-library/blob/e825065c46744fe6ce9c849b5c89cf54dd67fbe3/components/bl-signature-pad-component/README.md
https://github.com/Backendless/ui-builder-library/blob/49888d7010132b23dece6cec2e5f5b74221a8cd4/components/bl-gallery-component/README.md